_docs/style/glossary.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Glossary
+layout: docs
+category: Style
```

Comment:
A glossary wouldn't go into the Style category, would it? I would think this would go under `Getting Started` or `Resources`.

Reply:
We're still not a hundred percent on where this should be placed or if it should be placed on the site.
Also, we don't seem to have a place for getting started content yet in the repository.

Reply:
Kk, if you want to add it to the eventual Getting Started section, we can easily set that up in this PR.

Reply:
It will be under resources, but there are some difficulties in doing that straight away. We need to build a family page for resources and move the content from the resources page that exists (and probably rewrite while we are at it).